templates/filters.html

```diff
@@ -45,10 +45,9 @@
                     </span>
                 </div>
                 <select id="feature" class="selectpicker sp2 col-sm-4" multiple data-selected-text-format="count > 2" data-live-search="true" title="Select Features">
-                    <option value="EC">Embedded Checkout</option>
-                    <option value="RS">Reserved Seating</option>
-                    <option value="RE">Repeating Event</option>
-                    <option value="CQ">Custom Questions</option>
+                    {% for feature in features %}
+                    <option value="{{ feature.code }}">{{ feature.name }}</option>
```

Comment:
I'm not sure but `features` its a dict, and each feature it's a dict too...
so to access to the feature code and name I think that you have to do:
`"{{ feature['code'] }}">{{ feature['name'] }}
`

Reply:
I agree with @arielaranda-eb .

Reply:
In html you cannot access a dict item by doing dict['key']. This raises a TemplateSyntaxError.